### ISSUE TITLE
continuous delivery broken

### ISSUE THREAD

**Issue by monperrus** [SpoonLabs/gumtree-spoon-ast-diff#143]
There were no automated release on Central since Jan 13.

**Comment by martinezmatias** [comment]
I think that it's related to the bump to Java 11.
The Travis' log says: 
`Skipping deployment: wrong JDK. Expected 'oraclejdk8' but was 'oraclejdk11'.`
I will try to fix it.

**Comment by martinezmatias** [comment]
```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-source-plugin:2.3:jar-no-fork (attach-sources) on project gumtree-spoon-ast-diff: Execution attach-sources of goal org.apache.maven.plugins:maven-source-plugin:2.3:jar-no-fork failed: Plugin org.apache.maven.plugins:maven-source-plugin:2.3 or one of its dependencies could not be resolved: The following artifacts could not be resolved: org.codehaus.plexus:plexus-archiver:jar:2.4.4, org.apache.commons:commons-compress:jar:1.5: Could not transfer artifact org.codehaus.plexus:plexus-archiver:jar:2.4.4 from/to central (https://repo.maven.apache.org/maven2): Connection reset -> [Help 1]

```

**Comment by martinezmatias** [comment]
It seems the new version was updated: https://search.maven.org/artifact/fr.inria.gforge.spoon.labs/gumtree-spoon-ast-diff (v1.30)

**Comment by monperrus** [comment]
Thanks for the fix Matias.

**Comment by monperrus** [comment]
CD seems fixed by #152
